source/sync/app-builder/local-to-sync.txt

```diff
@@ -48,35 +46,34 @@ To get started creating the backend app, follow the steps in
 Enable Authentication
 ~~~~~~~~~~~~~~~~~~~~~
 
-Atlas Device Sync requires authenticated users. Atlas App Services provides
+Device Sync requires authenticated users. Atlas App Services provides
 many different :ref:`Authentication Providers <authentication-providers>` such
-as email/password and OAuth; once you have configured one or more providers,
-you can then control exactly who has access to which data.
+as email/password and OAuth. Once you have enabled and configured authentication,
+you have full control over who has access to which data.
 
-To set up an authentication provider, choose one of the
-providers and follow the configuration steps for the provider(s)
-of your choice.
+To learn more and get started, see 
+:ref:`Authentication Providers <authentication-providers>`.
 
-Choose a Sync Mode
-~~~~~~~~~~~~~~~~~~
+Enable Sync
+~~~~~~~~~~~
 
-There are two Sync Modes: Flexible Sync and the older :ref:`Partition-Based Sync
-<partition-based-sync>`. We recommend using Flexible Sync.
+In your App Services App, configuring Sync is the final step. When you set up 
+Sync, you enable authenticated users to have online access to their current data.
+While offline, users can work with the most-recent data, but their changes
+won't sync until they're back online.
 
 With Flexible Sync, clients synchronize subsets of data based on queries to
-**queryable fields**. When configuring Flexible Sync, you decide which fields
-clients can query upon. Development Mode allows App Services to deduce the
-queryable fields based on your queries. To learn more, follow the steps in
+queryable fields. When configuring Flexible Sync, you decide which fields
+clients can query on. To enable Flexible Sync, follow the steps in
 :ref:`<enable-flexible-sync>`.
 
 Update the Client App
 ---------------------
 
 Now that you have the backend set up for Device Sync, you need to make a few 
-changes in your client app. Follow the steps in the Quick Start for your 
-language/platform:
+changes to your client app code. Follow the steps to set up Flexible Sync in the 
+Quick Start for the language/platform you are using:
 
-- :ref:`<java-client-quick-start-sync>`
```

Comment:
? why deleted? and we should add link to kotlin page: :ref:`<kotlin-client-quick-start>`

Reply:
Because the Java quickstart is for PBS, not Flex Sync.